src/ai/ai.cpp

```diff
@@ -17,6 +17,10 @@ ShipAI::ShipAI(CpuShip* owner)
     weapon_direction = EWeaponDirection::Front;
 
     update_target_delay = 0.0;
+
+    short_range = owner->getShortRangeRadarRange();
+    long_range = owner->getLongRangeRadarRange();
+    relay_range = long_range * 2.0f;
```

Comment:
getting this once means if a script changes it on the fly the AI won't be aware of it.